gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Mon Aug 24 15:14:03 PDT 2015
```

Comment:
It's counter-intuitive.  According to the wrapper plugin documentation, this should go to the source repository.  This is so that anyone checking out the code can run `./gradlew clean build` without having Gradle installed.